docs/docs/en/guide/resource/configuration.md

```diff
@@ -80,8 +80,8 @@ data.basedir.path=/tmp/dolphinscheduler
 # resource view suffixs
 #resource.view.suffixs=txt,log,sh,bat,conf,cfg,py,java,sql,xml,hql,properties,json,yml,yaml,ini,js
 
-# resource storage type: HDFS, S3, OSS, GCS, ABS, NONE
-resource.storage.type=NONE
+# resource storage type: LOCAL, HDFS, S3, OSS, GCS, ABS
+resource.storage.type=LOCAL
```

Comment:
My bad, seems I forget to change to docs when I use `LOCAL` as default

Reply:
Never mind, should always say thanks to everyone who help maintain the doc :D